output_template/python/usb_camera_demo.py

```diff
@@ -79,6 +80,32 @@ def add_class_label(canvas,
     cv2.putText(canvas, text, dl_corner, font, font_scale, font_color, line_type)
 
 
+def label_to_color_image(results, colormap):
+    """Adds color defined by the dataset colormap to the label.
+
+    Args:
+        results: A 2D array with integer type, storing the segmentation label.
+        length: Single value with int type. The number of classes.
+
+    Returns:
+        result: A 2D array with floating type. The element of the array
+            is the color indexed by the corresponding element in the input label
+            to the PASCAL color map.
```

Comment:
Maybe this is not `PASCAL`

Reply:
oops, fixed it with commit 7c71b86 as well